_includes/header.html

```diff
@@ -63,28 +63,33 @@
 		                            </a>
 		                            <ul>
 		                                <li>
-		                                    <a href="#">
+		                                    <a href="https://fossasia.org/">
 		                                        Main Site
 		                                    </a>
 		                                </li>
 		                                <li>
-		                                    <a href="#">
+		                                    <a href="http://blog.fossasia.org/">
 		                                        Blog
 		                                    </a>
 		                                </li>
 		                                <li>
-		                                    <a href="#">
-		                                        Events
-		                                    </a>
-		                                </li>
-		                                <li>
-		                                    <a href="https://gci16.fossasia.org">
-		                                        Google Code-In 2016
-		                                    </a>
+											<a href="#">Events</a>
```

Comment:
Please correct the indentation of this block.

Reply:
I just add multi level menu for event list